reefd/main.go

```diff
@@ -0,0 +1,124 @@
+package main
+
+import (
+	"database/sql"
+	"flag"
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+	_ "github.com/mattn/go-sqlite3"
```

Comment:
3rd party import in a separate group.

this import is actually special, so you might want to separate this out further.

Reply:
done